web_sdk/web_engine_tester/lib/static/host.css

```diff
@@ -2,3 +2,7 @@ body {
   margin: 0;
   padding: 0;
 }
+
+iframe {
+  border: none;
```

Comment:
Good call!

Reply:
I hope I don't need borders for other tests :P